modules/vmseries/variables.tf

```diff
@@ -90,6 +90,19 @@ variable "password" {
   type        = string
 }
 
+variable "ssh_key_path" {
```

Comment:
How about not adding a variable for a single key, but rather a set of key data - having only an option for a single key is rather a development/demo use case, in real world there will be multiple users. That's actually how it's done in GCP VM-Series module.

Reply:
For the moment the code is supporting only a single account. Changing it to setup multiple accounts - that would be a conceptual change.